core/src/main/java/org/hibernate/ogm/compensation/impl/InvocationCollectingGridDialect.java

```diff
@@ -97,24 +98,14 @@ public void executeBatch(OperationsQueue queue) {
 			while ( operation != null ) {
```

Comment:
Now that batching is always enabled, I think we could unify these different operation object hierarchies?